From 626dcc7b1b6568abaf8cc633afb366a396e92695 Mon Sep 17 00:00:00 2001
From: Shreenidhi Shedi <sshedi@vmware.com>
Date: Sun, 12 Feb 2023 16:17:02 +0000
Subject: [PATCH] Fix implicit declaration warning

Gcc throws an implicit declaration for 'pthread_setname_np'
function. Define _GNU_SOURCE in source file to fix it.

Signed-off-by: Shreenidhi Shedi <sshedi@vmware.com>
---
 synce_port_ctrl.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/synce_port_ctrl.c b/synce_port_ctrl.c
index 788bf1a..454d67a 100644
--- a/synce_port_ctrl.c
+++ b/synce_port_ctrl.c
@@ -7,6 +7,7 @@
  * @note SPDX-FileCopyrightText: Copyright 2022 Intel Corporation
  * @note SPDX-License-Identifier: GPL-2.0+
  */
+#define _GNU_SOURCE
 #include <stdlib.h>
 #include <limits.h>
 #include <pthread.h>
